docs/source/features/version_control/squash_and_merge.rst

```diff
@@ -40,3 +44,43 @@ Or ``checkout`` to the target_branch first. In this case, the current branch is
     #. "override": the squashed branch will override the target branch;
 
     #. "skip": keep the origin branch.
+
+Get, list or delete
+===================
+
+.. literalinclude:: ../../../../docs/code/squash_and_merge.py
+      :language: python
+      :start-after: """Get, List and Delete"""
+      :end-before: """"""
+
+Get SquashAndMergeJob info
+==========================
+
+.. literalinclude:: ../../../../docs/code/squash_and_merge.py
+      :language: python
+      :start-after: """Get Job Info"""
+      :end-before: """"""
+
+.. note::
+    If the SquashAndMergeJob is successfully completed, the result will be a :class:`~tensorbay.client.struct.Draft`.
+
+Update
+======
+
+The SquashAndMergeJob information can be updated by :func:`~tensorbay.client.job.Job.update`. Note that if the ``until_complete`` is
```

Comment:
I think we should mention that the program will be blocked if the `until_complete` is `True`.
and there is no need to mention the interval time.